indexer-selection/src/test.rs

```diff
@@ -112,10 +92,8 @@ proptest! {
 fn sensitivity_seconds_behind() {
     let candidates = [
         Candidate {
-            indexer: hex!("0000000000000000000000000000000000000000").into(),
-            deployment: hex!("0000000000000000000000000000000000000000000000000000000000000000")
-                .into(),
-            url: "https://example.com".parse().unwrap(),
+            id: hex!("0000000000000000000000000000000000000000").into(),
```

Comment:
I used the `Address` type here to avoid changing these tests too much. But, if you prefer, I can use a simple `u64` type as the ID.

Reply:
I think for tests, it would be nicer to just use a u64 (less noise)

Reply:
Solved in https://github.com/edgeandnode/candidate-selection/commit/f23ba4e46a59c99348b8c435f8f3a774c9914de1